projects/ui-components/lf-metadata/lf-field-template-container/lf-field-template-container.component.ts

```diff
@@ -448,6 +447,7 @@ export class LfFieldTemplateContainerComponent extends LfFieldContainerDirective
         this.templateErrorMessage = this.TEMPLATE_HAS_FAILED_TO_LOAD;
         console.error('getTemplateFieldsAsync failed: ' + err?.message ?? err?.title ?? '');
         this.templateState = TemplateState.HAS_ERROR;
+        this.allFieldInfos = [];
```

Comment:
In this case, maybe this should go at the beginning.. Like clearing it before loading a new one.. (maybe this should be at the beginning updateTemplateFieldsAsync)